### PR TITLE
Add read and write file content actions

### PR DESCRIPTION
**Pull request by yoadev**

### CODE REVIEW

**Comment by Gui13** [comment]
LGTM

File: actions/file_get_content.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        # Get the project with id == project_id
+        project = gl.projects.get(project_id)
+
+        f = project.files.get(file_path, ref=branch)
+
+        if bool(decode_encoding):
+            return (True, base64.b64decode(f.content).decode(decode_encoding) )
+        else:
+            # get the base64 encoded content
+            return (True, f.content)
```
Comment (yoadev) [inline actions/file_get_content.py]:
```suggestion
            return (True, base64.b64decode(f.content))
```

**Review by fredgate — COMMENTED** [review]
LGTM

File: actions/file_get_content.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        # Get the project with id == project_id
+        project = gl.projects.get(project_id)
+
+        f = project.files.get(file_path, ref=branch)
+
+        if bool(decode_encoding):
+            return (True, base64.b64decode(f.content).decode(decode_encoding) )
+        else:
+            # get the base64 encoded content
```
Comment (fredgate) [inline actions/file_get_content.py:32]: the comment may no longer be entirely appropriate

**Pull request merged**